static/sass/_snapcraft_p-tabs.scss

```diff
@@ -8,6 +8,21 @@
     &__list {
       padding-right: 0;
       width: auto;
+      margin-bottom: -1px !important;
```

Comment:
Seems that `!important` is needed only because we add `u-no-margin--bottom`. No need to have `u-no-margin` if we set margin ourselves.

Reply:
I'm also worried that we are changing built in vanilla classes with custom styles here (`p-tabs`, `p-tabs__list`) that will be affected in other places as well - like on channel map (where we probably don't want `margin-bottom: -1px`.

Reply:
Yeah, I agree. There's a few bits coming up that use tabs. I've changed it to be `.p-tabs-bordered` and added an extra class for `tab_group`